src/components/organisms/news/news.html

```diff
@@ -0,0 +1,51 @@
+<div class="news">
+  <div class="news-and-events-title">
+    <h2>News<span class="caption"> お知らせ</span></h2>
+  </div>
+
+  <ul class="news-contents">
+    <a href="#" class="news-content-link">
+      <li class="news-content">
```

Comment:
> HTML の `<li>` 要素は、リストの項目を表すために用いられます。この要素は、その項目が属する順序付きリスト ([`<ol>`](https://developer.mozilla.org/ja/docs/Web/HTML/Element/ol))、順序なしリスト ([`<ul>`](https://developer.mozilla.org/ja/docs/Web/HTML/Element/ul))、メニュー ([`<menu>`](https://developer.mozilla.org/ja/docs/Web/HTML/Element/menu)) のいずれかの子要素として配置する必要があります。

[MDN](https://developer.mozilla.org/ja/docs/Web/HTML/Element/li)にはこのように書かれているので、liはulの直下に配置するようにしてください！
```suggestion
  <ul class="news-contents">
      <li class="news-content">
        <a href="#" class="news-content-link">
```
こんな感じになるかと思います。（scssの修正も必要になるかもです）